assets/js/components/ecosystems/ResultsPage.jsx

```diff
@@ -13,11 +13,16 @@ import {
   Row
 } from 'react-bootstrap'
 
+import ReactGA  from 'react-ga'
+
 import CandidateMatchCandidate from './../organisms/CandidateMatchCandidate.jsx'
 import SurveyCompletionIndicator from './../organisms/SurveyCompletionIndicator.jsx'
 import ElectionDayReminder from './../ecosystems/ElectionDayReminder.jsx'
 import LoadingIndicator from './../atoms/LoadingIndicator.jsx'
 
+const GOOGLE_ANALYTICS = process.env['GOOGLE_ANALYTICS'];
```

Comment:
I'm not sure what the best practice is.  I think a not-terrible thing would be to have env vars we're not concerned about being discoverable by a determined person served up by a js module.

So `import { GOOGLE_ANALYTICS } from clientEnvVars` or something like that.